modules/packages/DistributedBag.chpl

```diff
@@ -254,6 +254,12 @@ module DistributedBag {
       return chpl_getPrivatizedCopy(unmanaged DistributedBagImpl(eltType), _pid);
     }
 
+    proc readWriteThis(ch: channel) throws {
```

Comment:
Please add a test for this behavior (https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/ContributorInfo.rst#add-new-tests)

Reply:
@ben-albrecht Can you point me to the directory where the tests for DistributedBags are done? I cannot find it at ```test/library/packages```

Reply:
Sure, they are in `test/library/packages/Collection`

Reply:
@ben-albrecht Kindly review.